### PR TITLE
feat(productType-export): Add productType.key to export

### LINKED ISSUE #17: Support product type keys
### Description
Please provide support for product type keys (https://dev.commercetools.com/http-api-projects-productTypes.html#set-key) in the product type export, so that we are not dependent from the "uuid" of the database

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
PR adds `productType.key` to exported CSV/XLSX file.
Resolves #17

#### TODO

- Tests
    - [ ] Unit
    - [x] Integration
    - [ ] Acceptance
- [ ] Documentation

### CODE REVIEW

File: tests/integration/product-type-export.spec.js [hunk 1]
```diff
@@ -577,7 +586,7 @@ test(`productType export module
 test(`productType export module
   should output a product types and an attributes file`, (t) => {
   t.timeoutAfter(25000) // 25s
-  before().then(() => {
+  before([null]).then(() => {
```
Comment (wizzy25) [inline tests/integration/product-type-export.spec.js]: why are you passing a `null` array here? Would passing in no parameter make any difference?

File: tests/integration/product-type-export.spec.js [hunk 2]
```diff
@@ -205,7 +205,7 @@ const mockAttributes = mockProductTypes.reduce((attributes, type) =>
 
 let OUTPUT_FOLDER
 
-const before = function setup () {
+const before = function setup (keys) {
```
Comment (wizzy25) [inline tests/integration/product-type-export.spec.js]: I don't really understand what this is supposed to do

Comment (junajan) [reply tests/integration/product-type-export.spec.js]: This setup function removes and then creates several productTypes with automatically generated keys. In my test, I wanted to test also a use case when the key is not set so I added this `keys` parameter and when creating productType I check if we have a key for this Nth productType and if it is null I delete that generated key.

File: tests/integration/product-type-export.spec.js (hunk 1 above)
Comment (junajan) [reply tests/integration/product-type-export.spec.js]: Later in the `before` function (which also creates test productTypes) is a condition which removes key from productType draft if the key from `keys` parameter for given productType is `null`. So with this change, I test use case when `productType.key` is not set.

**Pull request merged**